docs/tutorial/view.md

```diff
@@ -0,0 +1,103 @@
+---
+category: tutorial
+order: 60
+menu-title: Updating editor UI
+---
+
+# Updating editor UI
+
+## Editor UI
+
+In the previous chapter of this tutorial, we created a command to highlight selected text. However, having to open a console and run a command is cumbersome and definitely not what editor users want.
+
+Let's update the editor interface to make this easier for them.
+
+The editor interface consists of two parts:
+
+* **toolbar** with buttons, dropdowns and other elements for performing various types of operations,
+* **editing view** for typing and modifying content.
+
+In this section, we'll focus on updating the toolbar.
+
+{@img assets/img/tutorial/ui.jpg Screenshot of the editor highlighting toolbar at the top and editing view at the bottom}
+
+## Creating a new button
+
+Let's import the `ButtonView` contructor in the `src/plugin.js` file:
+
+```js
+import { ButtonView } from 'ckeditor5/src/ui';
+```
+
+Then add the following code at the bottom of the `Highlight` method:
+
+```js
+editor.ui.componentFactory.add( 'highlight', ( locale ) => {
+	const button = new ButtonView( locale );
+	const command = editor.commands.get( 'highlight' );
+
+	button.set( {
+		label: editor.t('Highlight'),
```

Comment:
```suggestion
	const command = editor.commands.get( 'highlight' );
	const t = editor.t;

	button.set( {
		label: t('Highlight'),
```
Static analyzer looks for `t()` calls that can't be called as methods. See [here](https://www.notion.so/Internationalization-8651007a8ae74693ad0cc8a1f66e1226?pvs=4#c02d4f157a5f48b1a46a5368268497db).